site/src/examples/wealth-and-health-of-nations/index.js

```diff
@@ -0,0 +1,223 @@
+                        'y': labelBox.y,
+                        'width': labelBox.width,
+                        'height': labelBox.height
+                    })
+                    .on({
+                        'mouseover': function() { label.classed('active', true); },
+                        'mouseout': function() { label.classed('active', false); },
+                        'mousemove': mousemove,
+                        'touchmove': mousemove
+                    });
+            });
+        }
+
+        d3.rebind(yearLabel, event, 'on');
+
+        return yearLabel;
+    };
+
+    example.chart = function() {
+        var event = d3.dispatch('yearChange');
+
+        var radiusScale = d3.scale.sqrt().domain([0, 5e8]).range([0, 40]),
+            colourScale = d3.scale.category10();
+
+        function income(d) { return d.income; }
+        function lifeExpectancy(d) { return d.lifeExpectancy; }
+        function name(d) { return d.name; }
+        function population(d) { return d.population; }
+
+        function pointArea(d) { return Math.pow(radiusScale(population(d)), 2) * Math.PI; }
+        function sort(a, b) { return population(b) - population(a); }
+        function colour(d) { return colourScale(d.region); }
+
+        var point = fc.series.point()
+            .xValue(income)
+            .yValue(lifeExpectancy)
+            .key(name)
+            .size(pointArea)
+            .decorate(function(selection) {
+                selection.enter()
+                    .attr('fill', colour)
+                    .append('title')
+                    .text(name);
+
+                // Ensure the points with the smallest population are on top
+                selection.sort(sort);
+            });
+
+        var yearLabel = example.yearLabel()
+            .on('yearChange', event.yearChange);
+
+        var cartesianChart = fc.chart.cartesian(d3.scale.log(), d3.scale.linear())
+            .xDomain([300, 1e5])
+            .yDomain([10, 85])
+            .margin({left: 30, bottom: 20})
+            .yOrient('left')
+            .xTicks(12, d3.format(',d'))
+            .plotArea(point)
+            .xLabel('income per capita, inflation-adjusted (dollars)')
+            .yLabel('life expectancy (years)')
+            .decorate(function(selection, data) {
+                var xAxisWidth = selection.select('.x-axis')
+                    .layout('width');
+
+                selection.select('.x-axis .label')
+                    .attr('transform', 'translate(' + xAxisWidth / 2 + ', -26)');
+
+                var yAxisHeight = selection.select('.y-axis')
+                    .layout('height');
+
+                selection.select('.y-axis .label')
+                    .attr('transform', 'rotate(-90) translate(' + yAxisHeight / 2 + ', 36)');
+
+                var plotAreaContainerEnter = selection.enter()
+                    .select('.plot-area-container');
+
+                plotAreaContainerEnter.append('g')
```

Comment:
> Sorry, I think my original comment was lost ... 

Ignore me - Github was doing that thing of collapsing my comment because it was on an old diff. Good to see that adding a label translation has simplified things.